ts/features/profile/components/ProfileDeleteSuccess.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { View } from "react-native";
+import { ContentWrapper } from "@pagopa/io-app-design-system";
+import { SafeAreaView } from "react-native-safe-area-context";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../../components/ui/FooterWithButtons";
+import { InfoBox } from "../../../components/box/InfoBox";
+import { H3 } from "../../../components/core/typography/H3";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
```

Comment:
These are components that could be taken from `io-app-design-system` package. You can see all components of DS in a playground under Developer menu in Profile screen.